docs/inputs.md

```diff
@@ -263,6 +263,19 @@ To set multiple inputs, say:
 inspec exec my_profile --input input_name1=input_value1 name2=value2
 ```
 
+If a CLI input value resembles a number, it will be converted to an Integer or Float. Scientific notation is not currently recognized.
+```yaml
+inspec exec my_profile --input integer_input=-11
```

Comment:
These examples are potentially confusing because the inputs are named by their types - this _might_ read like we are typing the inputs here. Suggest using pretend real names like `amplifier_volume` `letters_array` `fruits_json`